test/src/org/exist/test/EmbeddedExistTester.java

```diff
@@ -1,140 +0,0 @@
-/*
```

Comment:
is it an option to have this 'deprecated' for a little time? removing it right away might be a bit harsh?